### PR TITLE
Widgets

### PR DESCRIPTION
**Pull request by chrisronline**
### Summary

We are adding a new Platform 2.0 primitive called `widgets`.

#### Documents
- [Product spec](https://corp.quip.com/RY5vA6C1hNhV/Product-Spec-Widgets)

See more in the canvas for [#feat-platform-widgets](https://slack.enterprise.slack.com/archives/C05J1STAXJ7)

### Requirements 

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/deno-slack-sdk/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've ran `deno task test` after making the changes.

### CODE REVIEW

File: src/widgets/types.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ export interface ISlackWidget {
 }
 
 export enum SlackWidgetDataMode {
-  PER_INSTALLER = "per_installer",
-  PER_USER = "per_user",
+  PER_INSTALLER = "PER_INSTALLER",
+  PER_USER = "PER_USER",
```
Comment (henrymingwu) [inline src/widgets/types.ts:10]: just updated these to the caps version if it's all the same to you, some slight convenience on my end and matches our API conventions more closely

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,9 +0,0 @@
-{
```
Comment (henrymingwu) [inline .vscode/settings.json:1]: I had originally removed this b/c I have a local extension that modifies this, but I see it's actually in `main` and we probably don't want to remove it.. I'll just deal with it myself for now and maybe ask the SDK folks if we really need it...

Comment (henrymingwu) [reply .vscode/settings.json]: added it back.

**Comment by filmaj** [comment]
Closing this abandoned feature.

**Pull request closed without merging**